create_appimage.sh

```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
+
+cd ..
+
 if [[ "$VSCODE_ARCH" == "x64" ]]; then
   # install a dep needed for this process
   sudo apt-get install desktop-file-utils
 
-  cd ..
-  export DOCKER_BUILD=1
```

Comment:
Does the latest version of the appimage utility auto detect it's running in a docker container now? This was required before.

Reply:
It seems so with [`grep docker /proc/1/cgroup >/dev/null && export DOCKER_BUILD=1 || true`](https://github.com/AppImage/pkg2appimage/blob/23e510b394fc28d24386e2eba076132a646f456c/functions.sh#L33).

But I'm having an issue the imagemagick's `convert` which are failing the packaging. And the error `Cannot mount AppImage, please check your FUSE setup.` is weird...